src/resource/resource.ts

```diff
@@ -176,6 +178,87 @@ export function isPodOwner(
   return podOwner === webId;
 }
 
+/**
+ * Get the URLs of Resources linked to the given Resource.
+ *
+ * Solid servers can link Resources to each other. For example, in servers
+ * implementing Web Access Control, Resources can have an Access Control List
+ * Resource linked to it via the `acl` relation.
+ *
+ * @param resource A Resource fetched from a Solid Pod.
+ * @returns The URLs of Resources linked to the given Resource, indexed by the key that links them.
+ * @since Not released yet.
+ */
+export function getLinkedResourceUrlAll(
+  resource: WithServerResourceInfo
+): internal_LinkedResourceUrls {
+  return resource.internal_resourceInfo.linkedResources;
+}
+
+/**
+ * What access the current user has to a particular Resource, and what access everybody has.
+ *
+ * Note that access for everybody is at the time of writing not returned by
+ * servers implementing Access Control Policies, so the `public` property is
+ * only available on servers that implement Web Access Control.
+ * @since Not released yet.
+ */
+export type EffectiveAccess = {
+  user: {
+    read: boolean;
+    append: boolean;
+    write: boolean;
+  };
+  public?: {
+    read: boolean;
+    append: boolean;
+    write: boolean;
+  };
+};
```

Comment:
Would this have its place in the  `interface.ts` file ?

Reply:
Yeah good point. We should probably consider at some point just getting rid of `interfaces.ts` altogether and just move the interfaces to the relevant modules, but as long as we haven't done that, best to be consistent: 911fa11f